portfolio/src/main/java/com/google/sps/servlets/ListCommentsServlet.java

```diff
@@ -45,7 +45,8 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
     SortDirection sortingDirection = sortingDirectionString.equals("descending") ? SortDirection.DESCENDING : SortDirection.ASCENDING;
 
-    Query query = new Query("Comment").addSort(sortingParameter, sortingDirection);
+    //  TODO: filter comments by user id. Make nickname be default author name.
```

Comment:
[Nit] Capitalize first word after TODO (should say "Filter")

Also, what is meant by nickname in this context?  Is that the same as the user's id?